packages/amplify-util-mock/package.json

```diff
@@ -50,19 +50,19 @@
     "which": "^2.0.2"
   },
   "devDependencies": {
-    "@aws-amplify/graphql-auth-transformer": "^2.1.0-cdkv2.2",
-    "@aws-amplify/graphql-default-value-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-function-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-http-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-index-transformer": "^1.1.0-cdkv2.2",
-    "@aws-amplify/graphql-maps-to-transformer": "^2.1.0-cdkv2.2",
-    "@aws-amplify/graphql-model-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-predictions-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-relational-transformer": "^1.1.0-cdkv2.2",
-    "@aws-amplify/graphql-searchable-transformer": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-transformer-core": "^1.1.0-cdkv2.1",
-    "@aws-amplify/graphql-transformer-interfaces": "^2.1.0-cdkv2.1",
-    "@aws-amplify/graphql-transformer-migrator": "^2.1.0-cdkv2.2",
+    "@aws-amplify/graphql-auth-transformer": "^2.1.0-cdkv2.3",
+    "@aws-amplify/graphql-default-value-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-function-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-http-transformer": "^1.1.0-cdkv2.2",
+    "@aws-amplify/graphql-index-transformer": "^1.1.0-cdkv2.3",
```

Comment:
is this correct?

Reply:
yes. we had 4 tag releases in data repo and the list of packages that got released was growing over time. hence last component is not in line.